src/reducers/historyReducer.js

```diff
@@ -50,7 +50,7 @@ export type HistoryAction = {
 export const initialState = {
   data: {},
   gasInfo: {
-    gasPrice: {},
+    gasPrice: null,
```

Comment:
Would it make more sense to store gas price per chain? instead of one chain gas price overriding other's chain price? Or this is not worth the hassle? wdyt?

Reply:
You're right, will update.